src/main/java/com/online/cinema/entity/moviesession/model/MovieSession.java

```diff
@@ -0,0 +1,31 @@
+package com.online.cinema.entity.moviesession.model;
+
+import com.online.cinema.entity.cinemahall.model.CinemaHall;
+import com.online.cinema.entity.movie.model.Movie;
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    private Movie movie;
+    @ManyToOne
```

Comment:
for this task it's optional, but remember that toOne relations have fetchType eager by default and it's not always quite good practice